README.md

```diff
@@ -1,5 +1,5 @@
-> [!WARNING]
-> UCX-Py is deprecated and will be phased out soon, version 0.45 (RAPIDS 25.08) is expected to be the last release. Projects that rely on UCX-Py are advised to migrate to [UCXX](https://github.com/rapidsai/ucxx) immediately.
+> [!CAUTION]
+> UCX-Py has been discontinued, version 0.45 (RAPIDS 25.08) was its last release and it will receive no further updates. Projects depending on UCX-Py are advised to migrate to [UCXX](https://github.com/rapidsai/ucxx) immediately. [UCXX documentation](https://docs.rapids.ai/api/ucxx/nightly/).
```

Comment:
```suggestion
> UCX-Py has been discontinued, version 0.45 (RAPIDS 25.08) was its last release and it will receive no further updates. Projects depending on UCX-Py are advised to migrate to [UCXX](https://github.com/rapidsai/ucxx) immediately. [UCXX documentation](https://docs.rapids.ai/api/ucxx/nightly/). For more details, see https://docs.rapids.ai/notices/rsn0050/.
```

I think we should add a link to the RSN. That's helpful in teaching RAPIDS users that that's a section of the docs they should watch for breaking changes like this.

I've been doing that elsewhere, for that reason:

* https://github.com/rapidsai/pynvjitlink/pull/155
* https://github.com/rapidsai/cuspatial/pull/1574